js/script.js

```diff
@@ -0,0 +1,72 @@
+// console.log("working!!!!");
+
+const APIKEY = "82f2229c255b1505c496b13203e48e52";
+const BASE = "https://api.openweathermap.org/data/2.5/forecast";
+const CITYID = "?id=2964574";
+const UNITS = "&units=metric";
+const URL = `${BASE}${CITYID}${UNITS}&APPID=${APIKEY}`;
+
+function getDay(longDate) {
+  let date = new Date(longDate);
+  let day = date.toString().substring(0, 3);
+  return day;
+}
+
+fetch(URL).then(response => {
```

Comment:
It would be a good idea to put this whole fetch thing inside a 
`document.addEventListener("DOMContentLoaded",  async function ()`

Then you have the advantage that 
1) You can place your script tag anywhere
and
2) As you are not at the top level you can use async, so you don't have to use the `then()` callback of fetch.